coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -32,6 +32,11 @@
                 <span v-else-if="isPending(item)">FILING PENDING</span>
                 <span v-else-if="isError(item)">FILING PENDING</span>
               </div>
+              <div class="filing-status-2" v-if="inProcessFiling === item.id && item.id !== undefined">
```

Comment:
You probably don't need the second check since the first check would already return false (null !== undefined).

Reply:
Let me check... I ran into the pending to-dos (tasks, not filings) being marked as processing but that might not be an issue anymore. I'll experiment.